content/api/metrics.apib

```diff
@@ -14,51 +14,56 @@ Metrics API data is retained for 6 months.
-| `count_spam_seed_private`        | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Private Seed messages delivered to the spam folder.    
-| `count_inbox_seed_virtual`       | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Virtual Seed messages delivered to the inbox.                                                               |
-| `count_spam_seed_virtual`        | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Virtual Seed messages delivered to the spam folder.    
-| `count_moved_to_inbox`           | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Panel messages delivered to the spam folder then moved to the inbox.                                |
-| `count_moved_to_spam`            | <a href="https://www.sparkpost.com/features/email-deliverability/"><span class="label label-warning">Deliverability Add-On</span></a> Panel messages delivered to the inbox then moved to the spam folder.                                |
+| Term                                          | Definition                                                                                          |
+|----------------------------------------------:|-----------------------------------------------------------------------------------------------------|
+| `count_targeted`                              | Messages successfully injected into SparkPost as well as rejected by it                             |
+| `count_injected`                              | Messages injected to or received by SparkPost                                                       |
+| `count_sent`                                  | Messages that SparkPost attempted to deliver, which includes both Deliveries and Bounces            |
+| `count_accepted`                              | Messages an ISP or other remote domain accepted (less Out-of-Band Bounces)                          |
+| `count_delivered`                             | Messages delivered                                                                                  |
+| `count_delivered_first`                       | Messages delivered on the first attempt                                                             |
+| `count_delivered_subsequent`                  | Messages delivered that required more than one delivery attempt                                     |
+| `count_rendered`                              | Total renderings of a message                                                                       |
+| `count_nonprefetched_rendered`                | Total nonprefetched renderings of a message                                                         |
```

Comment:
I'm not a good englisher. Does it matter if non has a hyphen after it, in the description? `non-prefetched`

Reply:
That's a tough question... Seems like it's [arbitrary](https://en.wiktionary.org/wiki/non-#:~:text=The%20prefix%20non%2D%20may%20be,British%20sport%2C%20is%20rare.))

Reply:
But anyway, corrected by [02b7278](https://github.com/SparkPost/developers.sparkpost.com/pull/521/commits/02b727816a649761a36a9c2a42ee689a9645f6b7)